src/content/docs/usage/workspaces.mdx

```diff
@@ -88,11 +88,34 @@ __Prerequisites__
 __Procedure__
 1.  Run the following command:
     ```shell
-    $ daytona delete
+    daytona delete
     ```
-2.  Select the desired Workspace for deletion in the list and hit `Enter`.
+2.  Mark Workspaces for deletion by highlighting them and pressing `x`.
```

Comment:
@osslate not sure I understand what `x` means here

Reply:
It means X on your keyboard. Run the command and you'll see